index.html

```diff
@@ -0,0 +1,67 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="styles.css" rel="stylesheet" type="text/css" />
+    <title>Coin Game</title>
+</head>
+
+
+<body>
+    <div>
+    <div class=container>
+    
+
+    <div><h1>COIN FLIP GAME</h1></div>
+
+    <div>
+        <img 
+            style="display: none" 
+            id="Himage" 
+            src="Heads.png">   
+    </div>
+
+    <div>
+        <img 
+            style="display: none" 
+            id="Timage" 
+            src="Tails.png">
+    </div>
+
+    <br>
+        <div class="buttonSize">
+            <button 
+            id="Click" 
+            onClick="flipCoin()">Click Here to Flip Coin
+        </button>
+    </div>
+    
+
+        <div>
+            <P 
```

Comment:
🔵 This isn't a problem now, but if you were using react, capitalizing the `p` tag here would cause issues.